lib/mini_stitch.rb

```diff
@@ -31,11 +31,17 @@ def self.status
   end
 
   class Configuration
-    attr_accessor :token, :client_id
+    attr_accessor :token, :client_id, :secondary_token, :secondary_client_id
 
     def initialize
       @token = nil
       @client_id = nil
+      @secondary_token = nil
+      @secondary_client_id = nil
+    end
+
+    def has_second_configuration? 
+      !secondary_token.nil? && !secondary_client_id.nil?
     end
```

Comment:
```suggestion
    def has_second_configuration? 
      secondary_token.present? && secondary_client_id.present?
    end
```

Reply:
Unfortunately `.present?` is a rails method and can not be used here.

Reply:
TIL. I've been working in Rails land for too long.